sources/packages/backend/libs/integrations/src/esdc-integration/e-cert-integration/e-cert-part-time-integration/_tests_/unit/e-cert-part-time-integration.service.spec.ts

```diff
@@ -0,0 +1,186 @@
+        stopFullTimeBCFunding: false,
+        courseLoad: 20,
+        applicationNumber: "2023000009",
+        documentNumber: 28,
+        disbursementDate: new Date("2023-11-03T00:00:00.000Z"),
+        documentProducedDate: new Date("2023-11-03T20:37:58.422Z"),
+        negotiatedExpiryDate: new Date("2023-11-03T00:00:00.000Z"),
+        schoolAmount: 0,
+        educationalStartDate: new Date("2023-09-15T00:00:00.000Z"),
+        educationalEndDate: new Date("2024-08-30T00:00:00.000Z"),
+        federalInstitutionCode: "DDDD",
+        weeksOfStudy: 51,
+        fieldOfStudy: 15,
+        yearOfStudy: 1,
+        completionYears: "12WeeksToLessThan1Year",
+        enrollmentConfirmationDate: new Date("2023-11-03T19:59:44.913Z"),
+        dateOfBirth: new Date("1973-01-31T00:00:00.000Z"),
+        lastName: "SHAH",
+        firstName: "RAM DEV",
+        addressLine1: "123 Gorge Rd E",
+        addressLine2: "",
+        city: "Victoria",
+        country: "Canada",
+        provinceState: "BC",
+        postalCode: "V1V1V1",
+        email: "simsthree@test.ca",
+        gender: "male",
+        ppdFlag: null,
+        maritalStatus: RelationshipStatus.Other,
+        studentNumber: "",
+        awards: [
+          {
+            valueType: DisbursementValueType.CanadaLoan,
+            valueCode: "CSLP",
+            valueAmount: 6554,
+            effectiveAmount: 6554,
+          },
+          {
+            valueType: DisbursementValueType.CanadaGrant,
+            valueCode: "CSGP",
+            valueAmount: 999,
+            effectiveAmount: 999,
+          },
+          {
+            valueType: DisbursementValueType.CanadaGrant,
+            valueCode: "CSPT",
+            valueAmount: 1800,
+            effectiveAmount: 1800,
+          },
+          {
+            valueType: DisbursementValueType.CanadaGrant,
+            valueCode: "CSGD",
+            valueAmount: 777,
+            effectiveAmount: 777,
+          },
+          {
+            valueType: DisbursementValueType.BCTotalGrant,
+            valueCode: "BCSG",
+            valueAmount: 333,
+            effectiveAmount: 333,
+          },
+        ],
+      },
+    ];
+
+    const [ecertRecord] = ecertRecords;
+    const disbursementAmount = ecertRecord.awards.filter(
+      (award) => award.valueCode === "CSLP",
+    )[0].effectiveAmount;
+    const totalBCGrantAmount = ecertRecord.awards.filter(
+      (award) => award.valueCode === "BCSG",
+    )[0].effectiveAmount;
+    const csgpPTAmount = ecertRecord.awards.filter(
+      (award) => award.valueCode === "CSPT",
+    )[0].effectiveAmount;
+    const csgpPDAmount = ecertRecord.awards.filter(
+      (award) => award.valueCode === "CSGP",
+    )[0].effectiveAmount;
+    const csgpPTDEPAmount = ecertRecord.awards.filter(
+      (award) => award.valueCode === "CSGD",
+    )[0].effectiveAmount;
+    const totalCanadaAndProvincialGrantsAmount =
+      totalBCGrantAmount + csgpPTAmount + csgpPDAmount + csgpPTDEPAmount;
```

Comment:
If the values are entirely under our control is there a need to calculate them dynamically?

Reply:
I thought it might be a good idea but I reverted to only values.